.github/workflows/python-ci.yml

```diff
@@ -24,6 +24,6 @@ jobs:
         pip install -r requirements.txt -r test_requirements.txt
         pip install -e .
     - name: Test with pytest (full)
-      if: github.ref == 'refs/heads/master'
+      if: github.ref != 'refs/heads/main'
       run: |
-        py.test
+        pytest
```

Comment:
I think this goes against what I said to you in PMs but if we are just running all the tests we do not need to have the ref check. I would however, recommend using the patterns we've established in other other repos with full and fast checks.